src/test/features/dashboard/routes/contact-them.ts

```diff
@@ -0,0 +1,55 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from 'test/routes/hooks'
+import 'test/routes/expectations'
+
+import { Paths } from 'dashboard/paths'
+
+import { app } from 'main/app'
+
+import * as idamServiceMock from 'test/http-mocks/idam'
+import * as claimStoreServiceMock from 'test/http-mocks/claim-store'
+import { checkAuthorizationGuards } from 'test/features/dashboard/routes/checks/authorization-check'
+
+const cookieName: string = config.get<string>('session.cookieName')
+
+const contactThemPage = Paths.contactThemPage.evaluateUri({ externalId: 'b17af4d2-273f-4999-9895-bce382fa24c8' })
+
+describe('Dashboard - Contact-them page', () => {
+  attachDefaultHooks(app)
+
+  describe('on GET', () => {
+    checkAuthorizationGuards(app, 'get', contactThemPage)
+
+    context('when user authorised', () => {
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor('1', 'citizen')
+      })
+
+      it('should return 500 and render error page when cannot retrieve address', async () => {
+        claimStoreServiceMock.rejectRetrieveClaimByExternalId('HTTP error')
+
+        await request(app)
+          .get(contactThemPage)
+          .set('Cookie', `${cookieName}=ABC`)
+          .expect(res => expect(res).to.be.serverError.withText('Error'))
+      })
+
+      context('when at least one claim issued', () => {
```

Comment:
you should do at least 3 different versions of this,
1. claimant
2. defendant
3. whichever with a correspondence address

and then assert the right sections show up

Reply:
We currently lack of global approach for testing this properly. As discussed this will be done as part of the spike that will address this: https://tools.hmcts.net/jira/browse/ROC-4211